api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -292,6 +292,7 @@ message CertificateValidationContext {
   // that if a CRL is provided for any certificate authority in a trust chain, a CRL must be
   // provided for all certificate authorities in that chain. Failure to do so will result in
   // verification failure for both revoked and unrevoked certificates from that chain.
+  // This behavior is disabled by configuring :ref:`crl_verify_all <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CertificateValidationContext.custom_validator_config>` false.
```

Comment:
```suggestion
  // The behavior of requiring all certificates to contain CRLs if any do can be altered by setting :ref:`crl_verify_all <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CertificateValidationContext.custom_validator_config>` false. If set to false, only the final certificate in the chain undergoes CRL verification.
```

I think ^ is a bit more clear. Also, the `crl` docs below need a similar update I think.

Reply:
Can we unify the terminology? https://www.openssl.org/docs/man1.0.2/man3/X509_VERIFY_PARAM_clear_flags.html
The doc uses "leaf certificate", we can use the same sth like `only_verify_leaf_cert_crl`.